src/order/well_founded.lean

```diff
@@ -80,26 +80,6 @@ begin
     exact hm y hy y_gt_m, },
 end
 
-lemma eq_iff_not_lt_of_le {α} [partial_order α] {x y : α} : x ≤ y → y = x ↔ ¬ x < y :=
-begin
-  split,
-  { intros xle nge,
-    cases le_not_le_of_lt nge,
-    rw xle left at nge,
-    exact lt_irrefl x nge },
-  { intros ngt xle,
-    contrapose! ngt,
-    exact lt_of_le_of_ne xle (ne.symm ngt) }
-end
```

Comment:
I'd keep this lemma, since it might be useful in other cases.

Reply:
Yes, please move it to `order.basic`

Reply:
Golfed and moved.